python/py-jsonschema/Portfile

```diff
@@ -9,7 +9,7 @@ revision            0
 categories-append   devel
 license             MIT
 supported_archs     noarch
-platforms           {darwin any}
+platforms           {darwin}
```

Comment:
why this change?

Reply:
Because it will install a different version of the port (hence different files) based on the OS version

Reply:
then the `platforms` field should be removed, as `darwin` (without the curly brackets since there is only one thing specified) is the default value.